stats/test/handler.test.js

```diff
@@ -353,17 +350,40 @@ describe('HTTP request handler', () => {
     })
   })
 
+  describe('GET /participants/scheduled-rewards/daily', () => {
+    it('returns daily scheduled rewards for the given date range', async () => {
+      await pgPools.stats.query(
+        'INSERT INTO daily_scheduled_rewards (day, participant_address, scheduled_rewards) VALUES ($1, $2, $3)',
+        ['2024-01-11', '0x20', '1']
+      )
+
+      const res = await fetch(
+        new URL(
+          '/participants/scheduled-rewards/daily?address=0x20&from=2024-01-11&to=2024-01-12',
+          baseUrl
+        ), {
+          redirect: 'manual'
+        }
+      )
+      await assertResponseStatus(res, 200)
+      const stats = await res.json()
+      assert.deepStrictEqual(stats, [
+        { day: '2024-01-11', scheduled_rewards: '1' }
+      ])
+    })
+  })
+
   describe('GET /miners/retrieval-success-rate/summary', () => {
     it('returns a summary of miners RSR for the given date range', async () => {
       // before the range
-      await givenRetrievalStats(pgPool, { day: '2024-01-10', minerId: 'f1one', total: 10, successful: 1 })
```

Comment:
Can we pass `pgPools.evaluate` as the first argument? For consistency with other places, e.g.

```js
 givenDailyParticipants(pgPools.evaluate
```